src/elements/element.rectangle.js

```diff
@@ -50,42 +50,76 @@ module.exports = function(Chart) {
 	}
 
 	Chart.elements.Rectangle = Chart.Element.extend({
-		draw: function() {
+		draw: function(isHorizontalBar) {
+
 			var ctx = this._chart.ctx;
 			var vm = this._view;
-
-			var halfWidth = vm.width / 2,
-				leftX = vm.x - halfWidth,
-				rightX = vm.x + halfWidth,
-				top = vm.base - (vm.base - vm.y),
-				halfStroke = vm.borderWidth / 2;
+			var x1,	x2,	y1,	y2;
+			var sign = {};
+			var borderWidth = vm.borderWidth;
+			var borderSkipped = (vm.borderSkipped !== undefined)? vm.borderSkipped: 'bottom';
+
+			if ((isHorizontalBar === undefined) || (!isHorizontalBar)) {
+				// bar
+				x1 = vm.x - vm.width / 2;
+				x2 = vm.x + vm.width / 2;
+				y1 = vm.y;
+				y2 = vm.base;
+				sign.x = 1;
+				sign.y = (y2 > y1) ? 1:-1;
+			} else {
+				// horizontal bar
+				x1 = vm.base;
+				x2 = vm.x;
+				y1 = vm.y - vm.height /2;
+				y2 = vm.y + vm.height /2;
+				sign.x = (x2 > x1) ? 1:-1;
```

Comment:
I thought it was in the ESLint rules, we could add it maybe? (@zachpanz88). Also, no space between the condition and the question mark and no need for the parenthesis around the condition: `sign.x = x2 > x1? 1 : -1;` :)